### ISSUE TITLE
default download list option

### ISSUE THREAD

**Issue by irbrad** [persepolisdm/persepolis#914]
**System Details:**
* Operating System: Any
* Distro(For GNU/Linux and BSD users): N/A
* Desktop Environment(For GNU/Linux and BSD users): N/A
* Persepolis Version: 3.2.0
* How do you install Persepolis? GitHub Release

**Issue Description and steps to reproduce:**
Can an option be added to change the default download list from single download to scheduled downloads?

**Comment by mclose90-cmd** [comment]
I don't believe Persepolis is maintained anymore. Try uGet or JDownloader instead.
